openfe/setup/atom_mapping/lomap_scorers.py

```diff
@@ -291,10 +291,11 @@ def creates_heterocyle(mol):
 
 def transmuting_methyl_into_ring_score(mapping: LigandAtomMapping,
                                        beta=0.1, penalty=6.0):
-    """Penalises ring forming
+    """Penalises having a non-mapped ring atoms become a non-ring
```

Comment:
```suggestion
    """Penalises having a non-mapped ring atoms become a non-ring group
```
Or something like that.